<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDTStudio</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="CDTProjectTabWidget" name="tabWidgetProject">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>灾害工程管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>灾害模型管理</string>
    </property>
    <addaction name="actionSampleData"/>
    <addaction name="actionTrainSVM"/>
    <addaction name="actionSelectSample"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>承载体目标提取</string>
    </property>
    <addaction name="action_AutoTargetDetection"/>
    <addaction name="action_InteractiveExtract"/>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>灾情信息统计输出</string>
    </property>
    <addaction name="action_AfterDisasterMap"/>
   </widget>
   <widget class="QMenu" name="menu_10">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
   <addaction name="menu_9"/>
   <addaction name="menu_10"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Normal</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidgetProject">
   <property name="windowTitle">
    <string>Project</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeViewObjects">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>保存所有</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="menuRecent">
   <property name="text">
    <string>最近打开文件</string>
   </property>
  </action>
  <action name="actionE">
   <property name="text">
    <string> 撤销</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>恢复</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>黏贴</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionSampleData">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icons/DataSource.png</normaloff>:/Icons/DataSource.png</iconset>
   </property>
   <property name="text">
    <string>灾害样本库</string>
   </property>
  </action>
  <action name="actionTrainSVM">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icons/train.ico</normaloff>:/Icons/train.ico</iconset>
   </property>
   <property name="text">
    <string>灾害模型训练</string>
   </property>
  </action>
  <action name="action_TimeSeriesImgChangeDetect">
   <property name="text">
    <string>基于多时相数据</string>
   </property>
  </action>
  <action name="action_BackgroundChangeDetect">
   <property name="text">
    <string>基于矢量本底数据</string>
   </property>
  </action>
  <action name="action_AutoTargetDetection">
   <property name="text">
    <string>目标自动识别</string>
   </property>
  </action>
  <action name="action_InteractiveExtract">
   <property name="text">
    <string>人机交互修改识别结果</string>
   </property>
  </action>
  <action name="action_PreDisasterMap">
   <property name="text">
    <string>灾前图</string>
   </property>
  </action>
  <action name="action_AfterDisasterMap">
   <property name="text">
    <string>灾后图</string>
   </property>
  </action>
  <action name="action_MergeMap">
   <property name="text">
    <string>叠加图</string>
   </property>
  </action>
  <action name="actionSelectSample">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icons/PBCD.png</normaloff>:/Icons/PBCD.png</iconset>
   </property>
   <property name="text">
    <string>样本选取</string>
   </property>
  </action>
  <zorder>dockWidgetProject</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CDTProjectTabWidget</class>
   <extends>QTabWidget</extends>
   <header>cdtprojecttabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
